apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -1043,8 +1046,8 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
                                 <Col span={24}>
                                     <Row gutter={BIG_HORIZONTAL_GUTTER}>
                                         <Col span={formFieldsColSpan}>
-                                            <Row gutter={SMALL_VERTICAL_GUTTER}>
-                                                <Col span={24}>
+                                            <Row gutter={[0, 0]}>
```

Comment:
I guess this property is zero by default
```suggestion
                                            <Row>
```